ansible/idr-ms-image-region.yml

```diff
@@ -0,0 +1,23 @@
+- hosts: "{{ idr_environment | default('idr') }}-omeroreadonly-hosts"
+  become: true
+
+  pre_tasks:
+    - name: get database host
+      set_fact:
+        database_server_url: >-
+          {{
+            hostvars[groups[idr_environment | default('idr') + '-database-hosts'][0]]
+            ['ansible_' + (idr_net_iface | default('eth0'))]['ipv4']['address']
+          }}
```

Comment:
As per the above, using `idr-omero-readonly.yml` would cut some redundancy as this is variable is populated in https://github.com/IDR/deployment/blob/e90627c95da65d0ca05a0e954acc79552da0c4bc/ansible/idr-omero-readonly.yml#L180-L187